frontend/src/Pages/AdminPage/ManageAssignmentPage/AssignmentListPage.jsx

```diff
@@ -233,20 +233,25 @@ export function AssignmentListPage() {
             className="mr-1"
             icon={<EditOutlined className="align-middle" />}
           />
-          <Button
-            className="mx-1"
-            disabled={record.state === ACCEPTED}
-            danger
-            icon={
-              <CloseOutlined
-                className={
-                  record.state === ACCEPTED
-                    ? "align-middle text-gray-300"
-                    : "align-middle"
-                }
-              />
-            }
-          />
+          <Link
+            to={`/admin/manage-assignment/delete-assignment/${record.id}`}
+            state={{ background: location }}
+          >
+            <Button
+              className="mx-1"
+              disabled={record.state === ACCEPTED}
+              danger
+              icon={
+                <CloseOutlined
+                  className={
+                    record.state === ACCEPTED
+                      ? "align-middle text-gray-300"
+                      : "align-middle"
+                  }
+                />
+              }
+            />
+          </Link>
           <Button
```

Comment:
Dont use link with button could be disabled, try to use onClick (refer HomePage)